### PR TITLE
Update admin_interface.html

### PR DESCRIPTION
**Pull request by blakegeelen**
added a place for inputs on extra buttons. changed value to placeholders. still need to update graph with input

### CODE REVIEW

File: test.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+      ]
+    },
+    options: {
+      legend: { display: false },
+      title: {
+        display: true,
+        text: 'Predicted world population (millions) in 2050'
+      }
+    }
+});
```
Comment (met4000) [inline test.js:20]: I can't see this being imported in any of the other files; is this a temp file?

Comment (blakegeelen) [reply test.js:20]: That was just a test. I will delete it before merging

**Comment by met4000** [comment]
If this is being replaced by #66, can this be closed?

**Comment by blakegeelen** [comment]
moved changes to updated branch so this branch is no longer needed

**Pull request closed without merging**